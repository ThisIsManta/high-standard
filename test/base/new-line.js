/* eslint-disable no-unused-vars */

console.log('a', 'b')
console.log(
  'a',
  'b'
)

const a = { a: 1, b: 1 }
const b = {
  a: 1,
  b: 1,
}

function f(a, b, c) {}

function g(a,
  b, c,
) {}

f(a,
  b,
)

g(a, b)

if ((a && b) ||
  (f && g)) {
  void a
}
