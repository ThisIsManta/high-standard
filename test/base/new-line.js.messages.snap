error 4:6 [@stylistic/comma-dangle] Missing trailing comma.
error 7:19 [@stylistic/object-property-newline] Object properties must go on a new line.
error 15:11 [@stylistic/function-paren-newline] Expected newline after '('.
error 15:12 [levitate/parameter-new-line] Expected a new line here.
error 16:6 [levitate/parameter-new-line] Expected a new line here.
error 19:2 [@stylistic/function-paren-newline] Expected newline after '('.
error 19:3 [levitate/parameter-new-line] Expected a new line here.
error 28:1 [levitate/consecutive-block-new-line] Expected an empty line before here.